examples/notebooks/Mapping_Molecules.ipynb

```diff
@@ -0,0 +1,4292 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a37fc0cd",
+   "metadata": {},
+   "source": [
+    "# Mapping Molecules\n",
+    "\n",
+    "In this notebook we will cover how to map molecules in different ways and look at some of the things we can do with them."
```

Comment:
Just some general remarks:
- f string in experiment name without variable
- water_chemical.molecules could be a dataclass (see above)
- Maybe use `print(json.dumps(water_chemical.molecules['water']['groups'], indent=4))` so the file can be read more easily
- I would abbreviate ADFs or not ADFS

looks really good besides these small changes!

Reply:
Good catch on the f strings and ADFS.  I will adjust them accordingly thank you.

Reply:
resolved in 836ea79